ops.h

```diff
@@ -29,6 +29,22 @@
 #include "hwy/contrib/thread_pool/thread_pool.h"
 #include "hwy/profiler.h"
 
+namespace gcpp {
+
+// __builtin_sqrt is not constexpr as of Clang 17.
+#if HWY_COMPILER_GCC_ACTUAL && defined(HWY_HAVE_SCALAR_BF16_OPERATORS) && \
```

Comment:
Make sure these condition checks allow building with only C++17 support (no constexpr sqrt). There's a lot of platforms where this comes up (eg i think OSX might still be on C++17) so want to make sure it still builds.

Alternatively I've seen some code examples of hand-rolled constexpr sqrt.

Reply:
Thanks for pointing to this. We actually only want the GCC check, that's sufficient for constexpr sqrt, everywhere else we treat it as non constexpr.